src/internal/query.ts

```diff
@@ -110,12 +107,12 @@ export class Query<T = any> {
     }
 
     /* Compile relations
-     */
-    if (this.relations.length) {
+     * TODO develop relations */
```

Comment:
please remove relations from `quickstart.md` and from examples

Reply:
there is also a method `relation` in `FilterableQuery` can you review and comment or maybe add a throw error ?